adobe_xd/lib/page_link.dart

```diff
@@ -30,11 +30,11 @@ enum LinkTransition {
 }
 
 class PageLinkInfo {
-  final Function pageBuilder;
+  final Function? pageBuilder;
```

Comment:
Does this really need to be nullable? It's leading to a lot of ! operators down the chain. Can this be `required`?

Reply:
It would need to be nullable from this constructor in `example`: https://github.com/jrc2139/xd-to-flutter-plugin/blob/b6e5f39512a5994b8e3ebc317ce601eb503631be/adobe_xd/example/lib/Gear.dart#L172